README.md

```diff
@@ -19,3 +19,27 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+## Answer
+
+Choosing the first element gives us a 50% chance to pick a good pivot. In order for median-of-three to be better, the odds of picking a good pivot must be higher for this.
+
+In order for a selected pivot to be a good pivot, it must create two partitions of a least size $3n/4$ or $3/4$ the size of the original array.
+
+There are three ways for a pivot to be, good, bad, or amazing. This gives us three cases for each, where each case is equally likely. This gives us 3^3 possibilities, as there are three options and three slots, first, median, and last. Meaning there are 27 possible combinations of pivots. 
+We can find the odds of each case happening easily.
+The cases are as follows bad = B, good = G, and amazing = A:
+(BBB) (BBG) (BBA) (BGB) (BAB) (BGG) (BGA) (BAA) (BAG)
+(GGG) (GGB) (GGA) (GBG) (GAG) (GBB) (GBA) (GAA) (GAB)
+(AAA) (AAG) (AAB) (AGA) (ABA) (AGG) (AGB) (ABB) (ABG)
+Looking at these, we can see that the median value is an A or G, amazing or good in 6 of the top row cases, 6 of the middle row cases, and 6 of the bottom row cases.
+So for 18/27 cases, the median is a good pivot value. In 66.66% of cases, median-of-three is the better option, making median-of-three better than picking just the first element.
```

Comment:
That doesn't sound quite right -- and I don't understand what an amazing pivot is. It can be either good or bad according to the definition we used.

Reply:
I misunderstood the slide, >3/4 is not a good pivot, I've changed my conclusion to reflect this.